File: nodejs-sequelize/model/rbac/organization.js
const {DataTypes, Model, Deferrable} = require('sequelize')
const {sequelize} = require('../../conf/db')
const {User} = require('./user')

class Organization extends Model {}

Organization.init({
    // attributes
    name: {
        type: DataTypes.STRING,
        allowNull: false
    },
    type: {
        type: DataTypes.STRING,
        defaultValue: 'customer'
    },
    pid: {
      type: DataTypes.INTEGER,
    }
}, {
  sequelize,
  modelName: 'organization',
  tableName: 'organization'
});

module.exports = {
  Organization
}

File: go-demo/model/organization.go
package model

import (
	"github.com/jinzhu/gorm"
)

type Organization struct {
	gorm.Model
	Name string `json:"name" gorm:"column:name"`
	Type string `json:"type" gorm:"column:type"`
	Pid uint `json:"pid" gorm:"column:pid"`
}